### ISSUE TITLE
Unable to open fulfill order action view

### ISSUE THREAD

**Issue by spencern** [reactioncommerce/reaction#2142]
Steps to reproduce

1. Clone `release-1.1.1`
2. reaction reset
3. meteor npm install
4. reaction run
5. setup stripe & enable all three default Flat Rate shipping options
6. Place Order from incognito window
7. Click into order from notifications panel
8. Attempt to fulfill order

No console errors were shown when attempting to fulfill order, but dashboard action panel appears empty and when clicking the `<-` back arrow throws the following error.

```
main.js:354 Uncaught TypeError: Cannot read property 'pop' of undefined
    at Object.popActionView (http://localhost:3000/app/app.js?hash=cefcfaa4796cc18f5ea7d77d168df51176564e34:66581:20)
    at Object.handleActionViewBack [as onClick] (http://localhost:3000/app/app.js?hash=cefcfaa4796cc18f5ea7d77d168df51176564e34:30053:12)
    at Button._this.handleClick (http://localhost:3000/app/app.js?hash=cefcfaa4796cc18f5ea7d77d168df51176564e34:39219:21)
    at Object.ReactErrorUtils.js.ReactErrorUtils.invokeGuardedCallback (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:119400:16)
    at executeDispatch (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:119180:21)
    at Object.executeDispatchesInOrder (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:119203:5)
    at executeDispatchesAndRelease (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:118587:22)
    at executeDispatchesAndReleaseTopLevel (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:118598:10)
    at Array.forEach (native)
    at forEachAccumulated (http://localhost:3000/packages/modules.js?hash=f8877f8ddab1183cef2c3299adca352ccd2d7707:119507:9)
```
Which is throwing an error from here:
```
 popActionView: function () {                                                                                         //
    var actionViewStack = Session.get("admin/actionView");                                                             //
    actionViewStack.pop();                                                                                             // 354
    Session.set("admin/actionView", actionViewStack);                                                                  //
    this.setActionViewDetail({}, {                                                                                     //
      open: false                                                                                                      // 358
    });                                                                                                                //
  },     
```

![image](https://cloud.githubusercontent.com/assets/1203639/25160260/e17c0ae4-2469-11e7-8064-6ef864109eab.png)

**Comment by aaronjudd** [comment]
I've been able to replicate when using a) no reaction.json b) clicking notification -> order.  The entire adminActionView history becomes corrupt. looks like 
``` "profile" : {
        "preferences" : {
            "reaction-orders" : {
                "orderListFilters" : "processing",
                "orderListSelectedOrder" : null
            },```

we need to test and ensure that this is a regression and not introduced in #2117

**Comment by spencern** [comment]
I've just confirmed that this issue is reproducible on `master`, so it seems like a bug that has existed for a few releases and is not introduced in 1.1.1.

**Comment by aaronjudd** [comment]
Closing in favor of #2136

**Comment by zenweasel** [comment]
Actually closing

**Comment by zenweasel** [comment]
I will have @michael-alade look at this issue now and reopen if it's not happening

**Comment by kieckhafer** [comment]
Please check on the `release-1.2` branch, this seems related to what I fixed with the notifications opening the Orders dashboard in the wrong window.

**Comment by zenweasel** [comment]
We have been on `release-1.2`

**Comment by michael-alade** [comment]
This issue has already been fixed on `release-1.2`.

**Comment by zenweasel** [comment]
This is not fixed. Reopening

**Comment by zenweasel** [comment]
Reopening. Ran into this while testing the dependency PR.

The key part of the duplication steps is "Click into order from notifications panel". It's going from that panel to the orders panel that is causing the problem

**Comment by zenweasel** [comment]
Specifically reopening this instead of #2136 per @spencern comment

**Comment by kieckhafer** [comment]
This was fixed at some point with #2280 , did you just see this occurring right now (in the last few hours), or has it been longer than that?

I just pulled in a router fix for a different issue ( #2476  ), perhaps that re-broke it:?

**Comment by zenweasel** [comment]
I just saw this two hours ago when I reported it.

**Comment by kieckhafer** [comment]
Ok... well it's highly possible these two things are related then... todays PR merge fixes a much more serious problem though so we want to keep that pulled in and re-fix this issue I guess.

**Comment by zenweasel** [comment]
Closed via #2530
